apps/src/music/views/KeyHandler.tsx

```diff
@@ -73,10 +72,6 @@ const KeyHandler: React.FunctionComponent<KeyHandlerProps> = ({
           reportKeyPress('toggle-headers');
           dispatch(toggleHeaders());
         }
-        if (event.key === 'b') {
```

Comment:
> @bencodeorg: Why is this no longer needed?

So originally the beat pad was a panel that popped up above the controls bar during playback, and we wanted a way to close it once opened. Now, it's been moved into the left controls box under the Run button and it doesn't make as much sense to hide/show it manually. There's also not a way for users to hit this unless they know the special `ui-keyboard-shortcuts-enabled` URL param